code/modules/mob/mob.dm

```diff
@@ -628,9 +628,30 @@ GLOBAL_LIST_INIT(slot_equipment_priority, list( \
 		return 1
 
 	face_atom(A)
-	var/list/result = A.examine(src)
+	var/list/result
+	if(client)
+		LAZYINITLIST(client.recent_examines)
+		var/ref_to_atom = A.UID()
+		var/examine_time = client.recent_examines[ref_to_atom]
```

Comment:
Since you're using a lazylist, you should use the existing lazy list defines from `lists.dm` to interface with them.

```suggestion
		var/examine_time = LAZYACCESS(client.recent_examines, ref_to_atom)
```